bootstrap/cmd/kfctl/cmd/apply.go

```diff
@@ -39,7 +45,19 @@ var applyCmd = &cobra.Command{
 		if resourceErr != nil {
 			return fmt.Errorf("invalid resource: %v", resourceErr)
```

Comment:
I believe we want to return a KFError everywhere.
Check out how the coordinator does it: https://github.com/kubeflow/kubeflow/blob/7d976050933d642069591b72138eb3c152224d08/bootstrap/pkg/kfapp/coordinator/coordinator.go#L79

Ideally, we could make a helper function `NewInternalKFError(e error, msg string)` in `pkg/apis/kferrors.go` to reduce boilerplate.